### PR TITLE
Feature/audit

### PR DESCRIPTION
**Pull request by kalepail**

### CODE REVIEW

**Comment by kalepail** [comment]
I changed the routing and naming of /heal a bit to use `ctrl-account`. /trust/heal doesn't make sense to me. We have a concept of controlled accounts and we should use it.

**Comment by kalepail** [comment]
@silence48 Has the README been updated where necessary?

**Comment by silence48** [comment]
> @silence48 Has the README been updated where necessary?

I need to update it some still. though was planning to make better deployment instructions as part of gitbook

> I changed the routing and naming of /heal a bit to use `ctrl-account`. /trust/heal doesn't make sense to me. We have a concept of controlled accounts and we should use it.

Yes ctrl-account makes more sense!

File: wrangler/src/@utils/trust.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        })
+      )
+      // add the operation to remove the old managedata entry
+      .addOperation(
+        Operation.manageData({
+          name: `turret.${removeSignerObj.key}`,
+          value: null,
+        })
+      )
+      .setTimeout(5 * 60)
```
Comment (kalepail) [inline wrangler/src/@utils/trust.js]:
```suggestion
      .setTimeout(0)
```

Comment (kalepail) [reply wrangler/src/@utils/trust.js]: Or permit a timestamp input with a timeout check

**Pull request merged**